packages/bruno-electron/src/ipc/collection.js

```diff
@@ -94,6 +94,28 @@ const registerRendererEventHandlers = (mainWindow, watcher, lastOpenedCollection
     }
   });
 
+  // update collection properties
```

Comment:
@thehorse2000 You don't need a new electron event. There is already one which handles the updates for Bruno Config file.

https://github.com/usebruno/bruno/blob/main/packages/bruno-app/src/components/CollectionSettings/index.js#L34-L42

Can you update the code to use that?